driver/syscall_table.c

```diff
@@ -361,6 +361,9 @@ const struct syscall_evt_pair g_syscall_table[SYSCALL_TABLE_SIZE] = {
 #ifdef __NR_openat2
 	[__NR_openat2 - SYSCALL_TABLE_ID0] =                    {UF_USED, PPME_SYSCALL_OPENAT2_E, PPME_SYSCALL_OPENAT2_X},
 #endif
+#ifdef __NR_execveat
+	[__NR_execveat - SYSCALL_TABLE_ID0] =                    {UF_USED | UF_NEVER_DROP | UF_SIMPLEDRIVER_KEEP, PPME_SYSCALL_EXECVEAT_E, PPME_SYSCALL_EXECVEAT_X},
+#endif
 };
 
 /*
```

Comment:
I think we need routing table values too:
`[__NR_execveat - SYSCALL_TABLE_ID0] = PPM_SC_EXECVEAT,`

Reply:
Good catch @FedeDP! Thank you very much, I have added `PPM_SC_EXECVEAT` both as enum field, and as syscall routing table value.